seed/challenges/02-data-visualization-certification/data-visualization-certificate.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "Claim Your Data Visualization Certificate",
+  "isComingSoon": true,
+  "order": 21,
+  "time": "5m",
+  "challenges": [
+    {
+      "id": "561add10cb82ac38a17513b3",
+      "title": "Claim Your Data Visualization Certificate",
+      "challengeSeed": [
+        {
+          "properties": ["isHonest", "isDataVisualizationCert"],
+          "apis": ["/certificate/honest", "/certificate/verify/data-visualization"],
+          "stepIndex": [1, 2]
+        }
+      ],
+      "description": [
+        [
+          "http://i.imgur.com/syJxavV.jpg",
+          "An image of our Front End Development Certificate",
```

Comment:
So is this a Frond End Development certification or Data Visualization? If second one then the entire Spanish translation is wrong too.